### PR TITLE
Fixed coinmarketcap link (issue #10)

### PR DESCRIPTION
**Pull request by adobryn**
Added link to coinmarketcap to home.component

### CODE REVIEW

File: alerts-ui/src/app/home/home.component.html [hunk 1]
```diff
@@ -12,8 +12,9 @@ <h2>{{'home.descripion' | translate}}</h2>
       <a href="https://bittrex.com/">BITTREX</a>,
       <a href="https://www.binance.com/">BINANCE</a>,
       <a href="https://bitso.com/">BITSO</a>,
+	  <a href="https://coinmarketcap.com/">COINMARKETCAP</a>,
       <a href="https://hitbtc.com/">HITBTC</a>,
-      <a href="https://www.kucoin.com/">KUCOIN</a>
+      <a href="https://www.kucoin.com/">KUCOIN</a>
```
Comment (AlexITC) [inline alerts-ui/src/app/home/home.component.html]: Why is this changed?

File: alerts-ui/src/app/home/home.component.html [hunk 2]
```diff
@@ -12,8 +12,9 @@ <h2>{{'home.descripion' | translate}}</h2>
       <a href="https://bittrex.com/">BITTREX</a>,
       <a href="https://www.binance.com/">BINANCE</a>,
       <a href="https://bitso.com/">BITSO</a>,
+	  <a href="https://coinmarketcap.com/">COINMARKETCAP</a>,
```
Comment (AlexITC) [inline alerts-ui/src/app/home/home.component.html]: Please keep the alignment.

**Review by AlexITC — CHANGES_REQUESTED** [review]
While the change looks good, you are including commits related to other pull request, please submit the changes for this issue only.

Thanks!

File: alerts-ui/src/app/home/home.component.html (hunk 1 above)
Comment (adobryn) [reply alerts-ui/src/app/home/home.component.html]: I first added a new link to the end, and then sorted them alphabetically

File: alerts-ui/src/app/home/home.component.html (hunk 2 above)
Comment (adobryn) [reply alerts-ui/src/app/home/home.component.html]: I'll fix it

File: alerts-ui/src/app/home/home.component.html (hunk 1 above)
Comment (AlexITC) [reply alerts-ui/src/app/home/home.component.html]: Please, remove the extra spaces to keep the only required line.

Comment (adobryn) [reply alerts-ui/src/app/home/home.component.html]: sure, I already fixed this locally. Now trying to split the branches in order to avoid mixed commits:)

Comment (AlexITC) [reply alerts-ui/src/app/home/home.component.html]: An approach is to use `git rebase`, there you could mark the extra commits to be deleted and squash the commits related to this PR, then, do `git push -f`.

**Review by AlexITC — APPROVED** [review]

**Comment by AlexITC** [comment]
Merged manually at 90e9f5d5afc927c36a3b55e40979d611f5905659.

Thanks.
